aries-site/src/components/cards/PreviewCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from 'grommet';
+
+export const PreviewImageCard = ({ children, ...rest }) => {
+  return (
+    <Box
+      height="small"
+      round="xsmall"
+      overflow="hidden"
+      style={{ position: 'relative' }}
+      {...rest}
+    >
+      {children}
+    </Box>
+  );
+};
+
+PreviewImageCard.defaultProps = {
+  background: 'background-contrast',
```

Comment:
Does this differ from the background colors of the svgs? If so, maybe we should be those match.